analysis/ceca.py

```diff
@@ -17,6 +18,11 @@
 import ray
 import scared
 
+ABS_PATH = os.path.dirname(os.path.abspath(__file__))
```

Comment:
I think it would be useful to have a comment here specifying why this is needed.

Reply:
Done.